### PR TITLE
Dataentry updates

### PR DESCRIPTION
**Pull request by major**
Simplify the is_raw method into a property to avoid using a callable.

Also adjust `test_get_content` to store the parsed JSON in a different
variable to avoid reusing `content`. That confused mypy a little.

### CODE REVIEW

File: src/cloudimagedirectory/connection/connection.py [hunk 1]
```diff
@@ -2,24 +2,23 @@
...
 class DataEntry:
     """Handles a file from the bucket."""
 
-    # TODO: Set Any to a specific type.
-    def __init__(self, filename: str, content: Any) -> None:
-        """Constructor for DataEntry class."""
-        self.filename = filename
-        self.content = content
+    filename: str
+    content: str
```
Comment (miyunari) [inline src/cloudimagedirectory/connection/connection.py:19]: How does this work? Isn't this field used in other places as a dictionary? :D

Comment (major) [reply src/cloudimagedirectory/connection/connection.py:22]: @miyunari so in a dataclass, you don't need an `__init__` for setting class arguments. You just specify them at the top of the class definition and it takes them in if they're provided.

Comment (miyunari) [reply src/cloudimagedirectory/connection/connection.py:22]: Oh, I meant `content` was a dictionary before, so of type `any` and now you changed it to `string`. Doesn't that cause trouble in other places, where I used `content` as a dict before? :)

Comment (major) [reply src/cloudimagedirectory/connection/connection.py]: So the issue was that `test_get_content` was re-using `content` which was typed as a string. That confused `mypy`. I fixed it and switched the `content` argument to be a dictionary or `None`.

**Comment by major** [comment]
@miyunari Test coverage went down 0.15% in the patch but improved overall. 🙃 That's where the red :x: is coming from.

**Pull request closed without merging**